pkg/apis/application/v1alpha1/types.go

```diff
@@ -2172,6 +2175,32 @@ func (proj AppProject) IsDestinationPermitted(dst ApplicationDestination) bool {
 	return false
 }
 
+// SetK8SConfigDefaults sets Kubernetes REST config default settings
+func SetK8SConfigDefaults(config *rest.Config) error {
```

Comment:
This function should be in `util/kube` package but unfortunately, we have a circular dependency . As soon as `gitops` engine branch is merged this case be fixed.